src/services/sanity.js

```diff
@@ -354,6 +377,7 @@ export async function fetchScheduledReleases(brand, { page = 1, limit = 10 }) {
 * Fetch content by a specific Railcontent ID.
 *
 * @param {string} id - The Railcontent ID of the content to fetch.
+ * @param {string} contentType - The document type of content to fetch
```

Comment:
Really nit picky, but there's an unnecessary space
```suggestion
* @param {string} contentType - The document type of content to fetch
```